lib/web_ui/test/html/paragraph/text_goldens.dart

```diff
@@ -13,29 +13,28 @@ import 'package:web_engine_tester/golden_tester.dart';
 import 'helper.dart';
 
 /// Class that controls some details of how screenshotting is made.
-///
-/// (For Googlers: Not really related with internal Scuba anymore)
-class EngineScubaTester {
-  EngineScubaTester(this.viewportSize);
+class EngineGoldenTester {
+  EngineGoldenTester(this.viewportSize);
 
-  /// The size of the browser window used in this scuba test.
+  /// The size of the browser window used in this golden test.
   final ui.Size viewportSize;
 
-  static Future<EngineScubaTester> initialize(
+  static Future<EngineGoldenTester> initialize(
       {ui.Size viewportSize = const ui.Size(2400, 1800)}) async {
     assert(() {
       if (viewportSize.width.ceil() != viewportSize.width ||
           viewportSize.height.ceil() != viewportSize.height) {
         throw Exception(
-            'Scuba only supports integer screen sizes, but found: $viewportSize');
+            'Gold only supports integer screen sizes, but found: $viewportSize');
       }
       if (viewportSize.width < 472) {
-        throw Exception('Scuba does not support screen width smaller than 472');
+        // TODO: Do we still need this check?
+        throw Exception('We do not support screen width smaller than 472');
```

Comment:
nit: s/We do not/Web engine test harness does not/

Reply:
This isn't applicable anymore, I removed this `if` block :)